README.md

```diff
@@ -1 +1,52 @@
-# Portfolio-setup-and-mobile-first
\ No newline at end of file
+# Portfolio-setup-and-mobile-first
+
+> Description the project.
+
+- set up a new repository and prepare it for development using best practices (e.g. linters).
+- build the first 2 sections of the mobile website using the template you chose in the figma.
+
+## Built With
+
+- Major languages:
+
+* HTML and CSS
+
+- Technologies used:
+
+* Linters
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
```

Comment:
- [ ] **[OPTIONAL]:** Please, consider removing this boilerplate text from your README! let's keep it clean and professional. 🤓

Reply:
Ok, thanks @anagudelogu